### PR TITLE
Added staging build config

### PR DESCRIPTION
**Pull request by Miggets7**

### CODE REVIEW

**Review by MartinaeyNL — APPROVED** [review]

**Comment by MartinaeyNL** [comment]
@Miggets7 checking with you if this can be merged? Would be nice if we have the video playback fix in. ;)

**Comment by DonWillems** [comment]
@pierrekil can press the squash and merge button if it works for the customer app

**Pull request merged**
